### PR TITLE
Add Windows installation

### PR DESCRIPTION
**Pull request by mabelzhang**
Partially addresses https://github.com/ignitionrobotics/docs/issues/117

Builds with warnings.

Only OGRE supported for now.

### CODE REVIEW

File: tutorials/02_install.md [hunk 1]
```diff
@@ -110,16 +149,57 @@ when building Ign Rendering OptiX plugin, edit
...
 
-
 ### Building from source
 
+#### Ubuntu
+
+Clone source code
+
+```
+# This checks out the `master` branch. You can append `-b ign-rendering#` (replace # with a number) to checkout a specific version
```
Comment (JShep1) [inline tutorials/02_install.md:159]:
```suggestion
# This checks out the `main` branch. You can append `-b ign-rendering#` (replace # with a number) to checkout a specific version
```

File: tutorials/02_install.md [hunk 2]
```diff
@@ -58,10 +69,38 @@ Install dependencies:
         libignition-plugin-dev
     sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 800 --slave /usr/bin/g++ g++ /usr/bin/g++-8 --slave /usr/bin/gcov gcov /usr/bin/gcov-8
 
-Clone source code
+#### Windows 10
+
+First, follow the [ign-cmake](https://github.com/ignitionrobotics/ign-cmake) tutorial for installing Conda, Visual Studio, CMake, etc., prerequisites, and creating a Conda environment.
```
Comment (JShep1) [inline tutorials/02_install.md:74]:
```suggestion
First, follow the [ign-cmake](https://github.com/ignitionrobotics/ign-cmake) tutorial for installing Conda, Visual Studio, CMake, and other prerequisites, and also for creating a Conda environment.
```

**Review by JShep1 — COMMENTED** [review]
Looks good, few nitpicks

File: tutorials/02_install.md [hunk 3]
```diff
@@ -30,6 +20,27 @@ Install Ignition Rendering:
...
+
+```
+conda create -n ign-ws
+conda activate ign-ws
+```
+
+Install:
+
+```
+conda install libignition-rendering<#> --channel conda-forge
```
Comment (JShep1) [inline tutorials/02_install.md:38]: `libignition-rendering` isn't currently in `conda-forge`, should we hold off on adding the Windows binary installation for now?

**Comment by mabelzhang** [comment]
Closing in favor of #196, where comments above have been addressed.

**Pull request closed without merging**